test/Source/Fixtures/DoctrineORMQuerySourceIterator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Test\Source\Fixtures;
+
+use Exporter\Source\DoctrineORMQuerySourceIterator as BaseDoctrineORMQuerySourceIterator;
+
+class DoctrineORMQuerySourceIterator extends BaseDoctrineORMQuerySourceIterator
```

Comment:
as this is a fixture I would use it this way:

```diff
- class DoctrineORMQuerySourceIterator extends BaseDoctrineORMQuerySourceIterator
+ class DoctrineORMQuerySourceIterator extends Exporter\Source\DoctrineORMQuerySourceIterator
```

and remove the use statement